CHANGELOG.md

```diff
@@ -0,0 +1,45 @@
+# Changelog LocationInsights
+
+## Table of Contents
+
+- [r1.1](#r11)
+
+**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
+
+# r1.1
+## Release Notes
+
+This release contains the definition and documentation of
+* most-frequent-location v0.1.0-rc.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.1
+
+## most-frequent-location v0.1.0-rc.1
+
+**most-frequent-location v0.1.0-rc.1 is the 1st release candidate of the version 0.1.0**
+
+- 0.1.0-rc.1 Most Frequent Location API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/LocationInsights/blob/r1.1/code/API_definitions/most-frequent-location.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/LocationInsights/blob/r1.1/code/API_definitions/most-frequent-location.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/LocationInsights/blob/r1.1/code/API_definitions/most-frequent-location.yaml)
```

Comment:
These links wouldn't work, correction (compare with README.md):

```suggestion
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/LocationInsights/r1.1/code/API_definitions/most-frequent-location.yaml&nocors)
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/LocationInsights/r1.1/code/API_definitions/most-frequent-location.yaml)
```

Hint: you can test the links by temporarily creating a branch `r1.1` (you would need to delete it before creating the release tag).